docker/Dockerfile.make

```diff
@@ -1,8 +1,10 @@
 # docker build --rm  -f docker/Dockerfile.server -t woodpeckerci/woodpecker-server .
 FROM golang:1.18-alpine as golang_image
-FROM node:16-alpine
+FROM node:18-alpine
 
-RUN apk add make gcc musl-dev
+RUN apk add --no-cache --update make gcc binutils-gold musl-dev
+
+RUN corepack enable
```

Comment:
This would be better added as:
```
... \
  && corepack enable
```
to previous RUN

Reply:
The rest of the file favors `&& \`, so I've applied that style (personally, I agree that `\`/`&& ...` is better, but I try not to violate file style).